src/ZombieDriver.php

```diff
@@ -125,7 +125,10 @@ public function visit($url)
   }
 });
 JS;
-        $this->server->evalJS($js);
+        $out = $this->server->evalJS($js);
+        if (!empty($out)) {
+            throw new DriverException(sprintf('Error when loading page %s: %s', $url, $out));
```

Comment:
The right fix would rather be to prefix the error output with `ZombieServer::ERROR_PREFIX` and then `evalJS` will throw the exception for us.

Reply:
Then you will not get the same degree of specialized error message. You will get a more general error message, and you will not readily see the url that it tried to visit.

You can also see that the same coding pattern is applied to the click function at line 719